playbooks/10 - SP - FortiSOAR Health Assessment/FortiSOAR Health Assessment.json

```diff
@@ -0,0 +1,334 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "FortiSOAR Health Assessment",
+    "aliasName": null,
+    "tag": null,
+    "description": "",
```

Comment:
@cs-rushikesh-mehakare  as discussed, Add suitable description  for all the playbooks

Reply:
This will be added in next PR